src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,28 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <>
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{post.title}</h3>
+
+        <p>
+          {' Posted by  '}
+
+          {post.user && (<UserInfo user={post.user} />)}
+        </p>
+      </div>
+
+      <p className="PostInfo__body">{post.body}</p>
+
+      <hr />
+
+      { post.comments.length === 0
```

Comment:
You can simplify your condition
```suggestion
      { !post.comments.length
```